### ISSUE TITLE
Feature: Support for wildcard in .ncurc

### ISSUE THREAD

**Issue by DerZyklop** [raineorshine/npm-check-updates#894]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

---------------------------

## Steps to Reproduce
Write a wildcard in your .ncurc like 
```
{
  "reject": [
    "@angular/*"
  ]
}
```

## Current Behavior
ncu still lists all `@angular/*` packages.

## Expected Behavior
Don’t list packages starting with `@angular/`.
In other words: Act like `ncu '/^(?!@angular/).*$/'`

**Comment by raineorshine** [comment]
Thanks for reporting! Could you paste the `ncu` output here? I am not able to reproduce on my end.

```sh
raine[894]% cat package.json                                                                                        
{
  "license": "MIT",
  "dependencies": {
    "@angular/cli": "11.0.0",
    "@angular/core": "11.0.0",
    "@angular/material": "11.0.0"
  }
}
raine[894]% ncu                                                                                                     
Checking /Users/raine/projects/ncu-issues/894/package.json
[====================] 3/3 100%

 @angular/cli       11.0.0  →  12.0.0
 @angular/core      11.0.0  →  12.0.0
 @angular/material  11.0.0  →  12.0.0

Run ncu -u to upgrade package.json
raine[894]% echo '{"reject": ["@angular/*"] }' > .ncurc                                                             
raine[894]% ncu                                                                                                     
Using config file /Users/raine/projects/ncu-issues/894/.ncurc
Checking /Users/raine/projects/ncu-issues/894/package.json

No dependencies.
raine[894]%
```

**Comment by raineorshine** [comment]
Closing due to inactivity

**Comment by DerZyklop** [comment]
Don’t know what was wrong. Tested it today. It’s working. I am sorry. Thank you for the good work.
